content/en/docs/reference/kubernetes-api/common-definitions/delete-options.md

```diff
@@ -71,7 +71,14 @@ DeleteOptions may be provided when deleting an API object.
 
   Whether and how garbage collection will be performed. Either this field or OrphanDependents may be set, but not both. The default policy is decided by the existing finalizer set in the metadata.finalizers and the resource-specific default policy. Acceptable values are: 'Orphan' - orphan the dependents; 'Background' - allow the garbage collector to delete the dependents in the background; 'Foreground' - a cascading policy that deletes all dependents in the foreground.
 
+- **ignoreStoreReadErrorWithClusterBreakingPotential** (boolean)
 
+By enabling this option, the user can perform an unsafe, and force delete of an undecryptable/corrupt resource. This option is behind a new ALPHA feature gate, and it is disabled by default. In order to use this option, the cluster operator must enable the feature by setting the command line option `--feature-gates=AllowUnsafeMalformedObjectDeletion=true`.
 
+A resource is considered corrupt if it can not be successfully retrieved from the storage due to a) transformation error e.g. decryption failure, or b) the object failed to decode. Normal deletion flow is attempted first, and if it fails with a 'corrupt resource' error then it triggers the unsafe and force delete.
 
+The default value for this option is 'false', this maintains backward compatibility. When this field is set to 'true', the fields 'dryRun', 'gracePeriodSeconds', 'orphanDependents', 'preconditions', and 'propagationPolicy' must be left unset.
 
+NOTE: The user must have the 'unsafe-delete-ignore-read-errors' permission on the resource being deleted.
+NOTE: force delete ignores finalizer constraints, and skips precondition checks.
+WARNING: this may break the workload associated with the resource being force deleted, if it relies on the normal deletion flow, so cluster breaking consequences may apply.
```

Comment:
maybe it would be better to have the warning first to make sure that users are aware of the impact that this option may have before telling them how to use it?

Reply:
I put the warning at the beginning of the section